.github/workflows/gsy-e_ci.yaml

```
@@ -21,9 +21,16 @@ jobs:
       env:
         prBody: ${{ steps.PR.outputs.pr_body }}
       run: |
-        echo "::set-output name=INTEGRATION_TESTS_BRANCH::$(echo -e $prBody | sed -n 's/.*\*\*INTEGRATION_TESTS_BRANCH\*\*=\([^ ]*\).*/\1/p')"
+        echo "::set-output name=PARSED_BRANCH::$(echo -e $prBody | sed -n 's/.*\*\*INTEGRATION_TESTS_BRANCH\*\*=\([^ ]*\).*/\1/p')"
```

I propose to use a more explicit name, in order to not get confused later (when we maybe have to add more branches to the template),
```suggestion
        echo "::set-output name=PARSED_INTEGRATION_TESTS_BRANCH::$(echo -e $prBody | sed -n 's/.*\*\*INTEGRATION_TESTS_BRANCH\*\*=\([^ ]*\).*/\1/p')"
```

---

The problem here is that this might need to be also set locally. As far as I see, this precedes the tox commands, therefore if we make this change, we will have to change tox to use PARSED_INTEGRATION_TESTS_BRANCH, which is not ideal for the local user IMO.

---

@spyrostz 
This variable (PARSED_BRANCH / PARSED_INTEGRATION_TESTS_BRANCH) is only "auxiliar", meaning that it is only used inside the git workflow. It serves to evaluate wether anything was parsed from the PR message, and then in a next step, INTEGRATION_TESTS_BRANCH is set according to wether PARSED_BRANCH is null or not.

It is not entirely clear for me why you say this might be needed on the tox file. The variable that is used on the tox file is INTEGRATION_TESTS_BRANCH, which remains unchanged.

---

@ERafaelMartinez that's my bad, I thought while reviewing that it would also pass through tox and would be used by it. Since it is only used by the CI configuration then all good from my side. Thanks for the clarification here!

---

No worries! Thanks for reviewing :)